vendor/github.com/mariomac/pipes/pkg/node/node.go

```diff
@@ -170,7 +171,7 @@ func (i *Start[OUT]) Start() {
 
 func (i *Middle[IN, OUT]) start() {
 	if len(i.outs) == 0 {
-		panic("Middle node should have outputs")
+		panic(fmt.Sprintf("Middle node should have outputs %v", i))
```

Comment:
Have you had an issue with this? This error comes when a node does not define any output node, or if the output node has been disabled.

For example, it might happen if the Kubernetes Decorator node is configured to send data to the OTEL traces or OTEL metrics nodes, but these are disabled, as in this PR.